packages/claim-backend/README.md

```diff
@@ -11,20 +11,21 @@ Apart from Node version 18, Claim Backend also requires:
-| DB_HOST          | Host used by PostgreSQL DB (Default: 127.0.0.1)                                                                                                      | false                     |
-| DB_DATABASE      | Name of Database of PostgreSQL DB (Default: claim-backend)                                                                                           | false                     |
-| DB_USERNAME      | Username of PostgreSQL DB (Default: claim-backend)                                                                                                   | false                     |
-| DB_PASSWORD_PATH | Absolute file path or file path relative to `docker-compose.yaml` that stores the password of database of PostgreSQL DB (Default: ./db_password.txt) | false                     |
-| DB_SSLMODE       | A flag to turn on SSL Connection on DB (Default: false)                                                                                              | false                     |
-| DB_SSL_CERT_PATH | Path to SSL Certificate (.crt or .pem) (Default: `null`)                                                                                             | When `DB_SSLMODE` is True |
-| DB_LOGGING       | A flag to turn on verbose logging of DB (Default: true)                                                                                              | false                     |
-| CORS_ORIGIN      | Accepted Origin, can be string or Regex (Default: \*)                                                                                                | false                     |
-| MERKLE_TREE_PATH | Path merkle-tree-details.json, generated by Tree Builder                                                                                             | true                      |
+| param                | Description                                                                                                                                          | Required                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------- |
+| BACKEND_HOST         | IP used by backend (Default: 127.0.0.1)                                                                                                              | false                     |
+| BACKEND_PORT         | Port used by backend (Default: 3000)                                                                                                                 | false                     |
+| BACKEND_LOGGER_LEVEL | Logger level in backend (trace/debug/info/warn/error/fatal) (Default: debug)                                                                         | false                     |
```

Comment:
let's set the default to be `info` so on production, we can use without changing